std_msgs/Header header

float32 yaw_rate_err_amp
uint8 yaw_rate_err_amp_invalid_flag
uint8 qualifier_yaw_rate
float32 yaw_rate
uint8 yaw_rate_invalid_flag
uint8 yaw_rate_event_data_qualifier